src/librbd/ImageWatcher.cc

```diff
@@ -366,8 +366,10 @@ void ImageWatcher<I>::notify_quiesce(const AsyncRequestId &async_request_id,
                              << dendl;
 
   ceph_assert(attempts > 0);
+  auto notify_response = new watcher::NotifyResponse();
```

Comment:
Nit: where is this freed?

Reply:
... still open?

Reply:
note, I passed it to the `unique_ptr` in the lambda context below. I suppose it should do the magic?

Reply:
Ack -- missed the generalized lambda capture